inc/classes/admin/settings/class-page.php

```diff
@@ -921,11 +923,32 @@ private function assets_section() {
 						'button_label' => __( 'Activate combine JavaScript', 'rocket' ),
 					],
 				],
+				'exclude_inline_js'  => [
+					'type'              => 'textarea',
+					'label'             => __( 'Excluded Inline JavaScript', 'rocket' ),
+					// translators: %1$s = opening <a> tag, %2$s = closing </a> tag.
+					'description'       => sprintf( __( 'Specify patterns of inline JavaScript to be excluded from concatenation (one per line). %1$sMore info%2$s', 'rocket' ), '<a href="' . esc_url( $inline_js_beacon['url'] ) . '" data-beacon-article="' . esc_attr( $inline_js_beacon['id'] ) . '" rel="noopener noreferrer" target="_blank">', '</a>' ),
+					'container_class'   => [
+						get_rocket_option( 'minify_concatenate_js' ) ? '' : 'wpr-isDisabled',
+						'wpr-field--children',
+					],
+					'placeholder'       => 'recaptcha',
+					'parent'            => 'minify_concatenate_js',
+					'section'           => 'js',
+					'page'              => 'file_optimization',
+					'default'           => [],
+					'sanitize_callback' => 'sanitize_textarea',
+					'input_attr'        => [
+						'disabled' => get_rocket_option( 'minify_concatenate_js' ) ? 0 : 1,
+					],
+				],
 				'exclude_js'             => [
 					'type'              => 'textarea',
 					'label'             => __( 'Excluded JavaScript Files', 'rocket' ),
-					'description'       => __( 'Specify URLs of JavaScript files to be excluded from minification and concatenation.', 'rocket' ),
-					'helper'            => __( 'The domain part of the URL will be stripped automatically.<br>Use (.*).js wildcards to exclude all JS files located at a specific path.', 'rocket' ),
+					// translators: %1$s = opening <a> tag, %2$s = closing </a> tag.
+					'description'       => sprintf( __( 'Specify URLs of JavaScript files to be excluded from minification and concatenation (one per line). %1$sMore info%2$s', 'rocket' ), '<a href="' . esc_url( $exclude_js_beacon['url'] ) . '" data-beacon-article="' . esc_attr( $exclude_js_beacon['id'] ) . '" rel="noopener noreferrer" target="_blank">', '</a>' ),
+					'helper'            => __( '<strong>Internal:</strong> The domain part of the URL will be stripped automatically. Use(.*).js wildcards to exclude all JS files located at a specific path.<br>
+					<strong>3rd Party:</strong> Use URL full path, including domain name, to exclude external JS.', 'rocket' ),
```

Comment:
Multi-line translation? I guess it will be fun when the indentation changes :s

Reply:
better to concatenate the lines separately?

Reply:
IMHO, anything else is better. Personally I'd do `__() . '<br/>' . __()`.